plugins/performance-lab/includes/site-health/object-cache/helper.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Helper functions used for Object Cache Support Info.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for Object Cache Info fields.
+ *
+ * @return array<string, array{label: string, value: string}> Fields.
+ * @since n.e.x.t
+ */
+function perflab_object_cache_supported_fields(): array {
+	return array(
+		'extension'        => array(
+			'label' => __( 'Extension', 'performance-lab' ),
+			'value' => perflab_get_cache_type(),
+		),
+		'multiple_gets'    => array(
+			'label' => __( 'Multiple gets', 'performance-lab' ),
+			'value' => wp_cache_supports( 'get_multiple' ) ? __( 'Enabled', 'performance-lab' ) : __( 'Disabled', 'performance-lab' ),
+		),
+		'multiple_sets'    => array(
+			'label' => __( 'Multiple sets', 'performance-lab' ),
+			'value' => wp_cache_supports( 'set_multiple' ) ? __( 'Enabled', 'performance-lab' ) : __( 'Disabled', 'performance-lab' ),
+		),
+		'multiple_deletes' => array(
+			'label' => __( 'Multiple deletes', 'performance-lab' ),
+			'value' => wp_cache_supports( 'delete_multiple' ) ? __( 'Enabled', 'performance-lab' ) : __( 'Disabled', 'performance-lab' ),
+		),
+		'flush_group'      => array(
+			'label' => __( 'Flush group', 'performance-lab' ),
+			'value' => wp_cache_supports( 'flush_group' ) ? __( 'Enabled', 'performance-lab' ) : __( 'Disabled', 'performance-lab' ),
+		),
+	);
+}
+
+/**
+ * Attempts to determine which object cache is being used as in wp-cli repository.
+ *
+ * Note that the guesses made by this function are based on the WP_Object_Cache classes
+ * that define the 3rd party object cache extension. Changes to those classes could render
+ * problems with this function's ability to determine which object cache is being used.
+ *
+ * This function was copied from WP-CLI.
+ *
+ * @link https://github.com/wp-cli/wp-cli/blob/0ca6d920123ac904c918d69181edc5071dc92c9d/php/utils-wp.php#L259-L331.
+ *
+ * @global bool $_wp_using_ext_object_cache Whether external object cache is being used.
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @return string Object cache type.
+ * @since n.e.x.t
+ */
+function perflab_get_cache_type(): string {
+	global $_wp_using_ext_object_cache, $wp_object_cache;
+
+	$message = '';
+
+	if ( ! empty( $_wp_using_ext_object_cache ) ) {
```

Comment:
Per #1219 let's avoid using `empty()` here. And it seems we can avoid using the `$_wp_using_ext_object_cache` private global by instead just using the `wp_using_ext_object_cache()` function:

```suggestion
	global $wp_object_cache;

	$message = '';

	if ( wp_using_ext_object_cache() ) {
```

Reply:
@westonruter will fix mentioned issues after 12 of August, as I just started holidays.

Reply:
Great. Enjoy!